### PR TITLE
Added a toggle to hide avatars in group chats

### PR DESCRIPTION
**Pull request by OzTamir**
The prefences code is based on how I saw that the "preferContactAvatars" feature was implemented.

The functionallity changes are made in CVComponentMessage.swift

### First time contributor checklist

- [x ] I have read the [README](https://github.com/WhisperSystems/Signal-iOS/blob/master/README.md) and [CONTRIBUTING](https://github.com/WhisperSystems/Signal-iOS/blob/master/CONTRIBUTING.md) documents
- [x ] I have signed the [Contributor Licence Agreement](https://whispersystems.org/cla/)

### Contributor checklist

- [ ] I'm following the [code, UI and style conventions](https://github.com/WhisperSystems/Signal-iOS/blob/master/CONTRIBUTING.md#code-conventions) - **This link is broken...**
- [ ] My commits are rebased on the latest master branch
- [ x] My commits are in nice logical chunks
- [x ] My contribution is fully baked and is ready to be merged as is
- [x ] I have tested my contribution on these devices:
 * iPhone 11, iOS 14.3

- - - - - - - - - -

### Description
**TL;DR - Added a toggle in Appearance settings that prevents profile pictures from being displayed in group chat.**

Background:
I've recently joined Signal as most of my friends decided that it is time to move on from WhatsApp. While the app has been great for me, I found that I had many UI annoyances that prevented me from completely making the switch. I've asked around and many of my friends agreed. After a few talks, I've decided to make use of the fact that Signal is an open source application and set out to add the features that would make me enjoy the UI and would allow me to make the switch.

This is the first of those. I don't like the UI experience of having a lot of small images cluttering my group chats, and so in this PR I've added the ability to hide the sender's avatar in group chat messages.

Technical:
Most of the changes were made in order to add the storage support for this setting. I don't know if there's an easier way to do so, but I just went ahead and replicated the code for the other toggle in the appearance pane (preferContactAvatars). The actual code for hiding the avatars is rather small and is based in CVComponentMessage.swift.

I'm here for any comments, and waiting to hear from you :)

### CODE REVIEW

**Comment by Imperiopolis-Signal** [comment]
Thanks for your contribution. At this time, this is not something our design team would like to support, but feel free to start a discussion on our community forum and we'll keep an eye on it.
